helmfile/values/fluentd-configmap.yaml.gotmpl

```diff
@@ -0,0 +1,263 @@
+aggregator:
+  configMaps:
+    fluentd.conf: |
+      # Prometheus Exporter Plugin
+      # input plugin that exports metrics
+      <source>
+        @type prometheus
+        port 24231
+      </source>
+
+      # TCP input to receive logs from the forwarders
+      <source>
+        @type forward
+        bind 0.0.0.0
+        port 24224
+      </source>
+
+      # HTTP input for the liveness and readiness probes
+      <source>
+        @type http
+        bind 0.0.0.0
+        port 9880
+      </source>
+
+      <match kubernetes.var.log.containers.fluentd-aggregator-**>
+        @type null
+      </match>
+      <match **>
+        @id output-s3
+        @type s3
+
+        aws_key_id "#{ENV['AWS_ACCESS_KEY_ID']}"
+        aws_sec_key "#{ENV['AWS_ACCESS_SECRET_KEY']}"
+        {{ if eq .Values.fluentd.forwarder.useRegionEndpoint true -}}
+        s3_endpoint {{ .Values.objectStorage.s3.regionEndpoint }}
+        force_path_style true
+        {{ else -}}
+        s3_region {{ .Values.objectStorage.s3.region }}
+        {{ end -}}
+        s3_bucket {{ .Values.objectStorage.buckets.scFluentd }}
+
+        path logs/%Y%m%d/${tag}/
+
+        <buffer tag,time>
+          @type file
+
+          path /opt/bitnami/fluentd/logs/buffers/s3
+          overflow_action block
+          total_limit_size 9GB
+
+          timekey 10m
+          timekey_wait 1m
+          timekey_use_utc true
+          chunk_limit_size 50m
+        </buffer>
+      </match>
```

Comment:
I only see S3 and nothing for gcs.
https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/fluentd-aggregator.yaml.gotmpl#L37-L49